Mathlib/Topology/Algebra/IsUniformGroup/Basic.lean

```diff
@@ -90,28 +90,28 @@ section LatticeOps
 variable [Group β]
 
 @[to_additive]
-lemma IsUniformInducing.uniformGroup {γ : Type*} [Group γ] [UniformSpace γ] [UniformGroup γ]
+lemma IsUniformInducing.isUniformGroup {γ : Type*} [Group γ] [UniformSpace γ] [IsUniformGroup γ]
     [UniformSpace β] {F : Type*} [FunLike F β γ] [MonoidHomClass F β γ]
     (f : F) (hf : IsUniformInducing f) :
-    UniformGroup β where
+    IsUniformGroup β where
   uniformContinuous_div := by
     simp_rw [hf.uniformContinuous_iff, Function.comp_def, map_div]
     exact uniformContinuous_div.comp (hf.uniformContinuous.prodMap hf.uniformContinuous)
 
 @[deprecated (since := "2024-10-05")]
-alias UniformInducing.uniformGroup := IsUniformInducing.uniformGroup
+alias UniformInducing.isUniformGroup := IsUniformInducing.isUniformGroup
```

Comment:
I think that we should add new deprecated aliases, not edit the old ones. Also, we should have both multiplicative and additive versions. Since `to_additive` doesn't play well with `deprecated` at the moment, I usually use the following workaround: first create a deprecated alias for the additive version, then use `@[to_additive existing, deprecate (since := ..)]` (in this order) for the multiplicative version.

Reply:
Thanks for the suggestion. Can you check I've done it the right way? If it is OK I will do the same elsewhere.